index.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const pino = require('pino')
+const getPrettyStream = require('pino/lib/tools').getPrettyStream
```

Comment:
This is a private API and not guaranteed to remain consistent.

Reply:
agree,... but the problem here is that we just not want to expose this function from pino itself. See https://github.com/pinojs/pino/pull/666

I think there is no good solution at this point, but this way we prevent modifications to core that are not wanted.

Reply:
I believe if you follow the whole conversation in that issue you will see that it is argued the PR is fine for Pino v5 and there is discussion about what to do for Pino v6.

Reply:
Dave seemed to be against in landing that in v5 at all. I’m happy to land it here instead. This module is using a ton of private stuff anyway, and it’ll likely break if we change some internals.

Reply:
🤷‍♂️

Reply:
yeah I'd rather keep it out so that it's contained. There is no winnable situation here - if we want to do is "right" we can expose symbols etc. as we do for integration with pino-http etc. But the only reason to go to that effort is if there are use cases beyond this very specific one, and I don't see any other reasons to integrate with getPrettyStream. Ergo, I'm fine with relying on the private path in this one specific cases because we control all aspects of this